src/pages/patientView/clinicalInformation/lib/clinicalAttributesUtil.js

```diff
@@ -207,21 +207,89 @@ function getSpanElementsFromCleanData(
     clinicalAttributesCleanDerived,
     cancerStudyId
 ) {
-    let spans = [];
-    return Object.keys(clinicalAttributesCleanDerived).map(key => {
-        let value = clinicalAttributesCleanDerived[key];
-        return (
-            <span
-                is
-                class="clinical-attribute"
-                attr-id={key}
-                attr-value={value}
-                study={cancerStudyId}
-            >
-                {value}
-            </span>
-        );
-    });
+    // Order patient clinical attributes
+    const getOrder = key => {
+        const map = {
+            SEX: 1,
+            GENDER: 1,
+            AGE: 2,
+            CANCER_TYPE: 3,
+            CANCER_TYPE_DETAILED: 4,
+            KNOWN_MOLECULAR_CLASSIFIER: 5,
+            HISTOLOGY: 6,
+            OS_STATUS: 7,
+            OS_MONTHS: 8,
+            DFS_STATUS: 9,
+            DFS_MONTHS: 10,
+        };
+
+        return map[key] != undefined ? map[key] : Number.MAX_VALUE;
+    };
+
+    // Ascending order; e.g. GENDER precedes OS_MONTHS
+    const compare = (a, b) => {
+        return getOrder(a) - getOrder(b);
+    };
+
+    // Styles before and after value
+    function stringBuilder(value, key) {
+        var builder = [];
+
+        const before = {
+            /* attributes with opening parenthesis */
+            OS_MONTHS: '\xa0(',
```

Comment:
@jtquach1 @inodb, i wonder if we can make this more general.    @inodb could we have a list of primary attributes and secondary attributes?  Secondary would always be wrapped in parens.   lastly, we would need a map of "formatters", e.g.   { OS_MONTHS:  `${s} months`   }.   In fact, the formatters could be values in both primary and secondary maps.

Reply:
another thought, @jtquach1 i think we should also make color configurable via data.  so, it should be struck from css too and represented in data structure.